packages/cyberstorm/src/components/Switch.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropsWithChildren, useCallback } from "react";
+import styles from "./componentStyles/Switch.module.css";
+import * as RadixSwitch from "@radix-ui/react-switch";
+
+export interface SwitchProps {
+  state: boolean;
+  switchCallback?: (state: boolean) => void;
+  disabled?: boolean;
+}
+
+export const Switch: React.FC<PropsWithChildren<SwitchProps>> = ({
+  state,
+  switchCallback = undefined,
```

Comment:
Passing undefined as the default value is pointless in JS since missing keys in objects are undefined, and missing function parameters are undefined.